docker-scout/Jenkinsfile

```diff
@@ -60,6 +60,21 @@ def dockerScoutAnalyze(String version) {
     sh "docker run -t -v ${env.WORKSPACE}/report/:/report/ ${dockerLoginAsEnv()} " +
         "${dockerScoutContainer()} cves ${axonivyEngineImageUrl(version)} " +
         "--ignore-base --only-fixed --locations --format sarif --output /report/sarif.json"
+
+    sh "docker run -t -v ${env.WORKSPACE}/report/:/report/ ${dockerLoginAsEnv()} " +
+        "${dockerScoutContainer()} sbom ${axonivyEngineImageUrl(version)} " +
+        "--format cyclonedx --output /report/bom.json"
+
+            withCredentials([string(credentialsId: 'dependency-track', variable: 'API_KEY')]) {
+              sh 'curl -v --fail -X POST https://api.dependency-track.ivyteam.io/api/v1/bom \
+                      -H "Content-Type: multipart/form-data" \
+                      -H "X-API-Key: ' + API_KEY + '" \
+                      -F "autoCreate=true" \
+                      -F "projectName=axonivy/axonivy-engine" \
+                      -F "projectVersion=' + version + '" \
```

Comment:
no longer work with uuid, instead if projectName and projectVersion. dependency-track allows multiple versions of a project.